stories/user-input.stories.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import UserInputApp from '../assets/js/components/user-input/UserInputApp';
+
+storiesOf( 'User Input', module )
+	.add( 'User Input Page', () => {
+		return <UserInputApp />;
```

Comment:
While this component doesn't use the datastore yet since it's just scaffolded, let's already wrap this in a `WithDataRegistry` component, to which we can pass a default registry created via `createTestRegistry()`.

Reply:
@felixarntz I couldn't find any references to `WithDataRegistry`, so I am assuming that you meant `withTestRegistry` 😄

Reply:
Oops, exactly 😆